core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts

```diff
@@ -207,11 +216,21 @@ export class ServiceLogsComponent implements OnInit {
 	onregSelected(event){
     this.FilterTags.notify('filter-Region',event);
     this.regSelected=event;
-   }
+	 }
+	 onAssetListSelected(event){
+		var asset_type_list=this.cache.get('assetList');
+		var fName = asset_type_list.friendly_name;
+		var index = fName.indexOf(event);
+		var asset = asset_type_list.env[index];
+		this.asset_type = event;
+		this.payload.asset_type=asset;
+		this.resetPayload();
+	 }
  
 	// onEnvSelected(env){
 
 	onEnvSelected(envt){
+		debugger
```

Comment:
remove debugger and console from this file

Reply:
removed